pkg/logs/input/eventlog/tailer.go

```diff
@@ -0,0 +1,81 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package eventlog
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/config"
+	"github.com/DataDog/datadog-agent/pkg/logs/message"
+	log "github.com/cihub/seelog"
+	"github.com/clbanning/mxj"
+)
+
+// Config is a event log tailer configuration
+type Config struct {
```

Comment:
I am not sure it's still worth using this structure since it just maps the `source.Config`

Reply:
It allows to modify the tailer config without altering the source config